### PR TITLE
Redirect News to annoucements

### LINKED ISSUE #2707: Redirect scratch news page to announcements forum
### Is your request related to a problem? Please describe.

No. Simply the page scratch.mit.edu/news has barely any content and is unused.

### Describe the solution you'd like

Redirect to the announcements forum

### Additional context

I have an untested version on my fork

### PR DESCRIPTION
**Pull request by Shluffy**
Resolves #2707 

### Changes

redirects `https://scratch.mit.edu/news` to the annoucements forum

### Reason for changes

open issue, cool to have

### Tests

none

### CODE REVIEW

File: addons/news-annoucements/userscript.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default async function ({ addon, global, console }) {
+  window.location.href = "https://scratch.mit.edu/discuss/5/";
```
Comment (FunctionalMetatable) [inline addons/news-annoucements/userscript.js:2]: Use `window.location.replace` instead

Comment (Shluffy) [reply addons/news-annoucements/userscript.js]: done

File: addons/news-annoucements/addon.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Redirect scratch.mit.edu/news to Announcements Forum",
+  "description": "Redirects the page scratch.mit.edu/news to the announcemnts forum.",
+  "tags": ["community"],
```
Comment (mxmou) [inline addons/news-annoucements/addon.json:4]: This should be an Easter egg addon (`"easterEgg"` tag).

**Review by mxmou — CHANGES_REQUESTED** [review]

File: addons/news-annoucements/userscript.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default async function ({ addon, global, console }) {
```
Comment (FunctionalMetatable) [inline addons/news-annoucements/userscript.js:1]: I don't think you need to declare `addon, global, console` as we don't actually use it in the addon itself.

Comment (Shluffy) [reply addons/news-annoucements/userscript.js:1]: I would leave it as is, just because better safe then sorry

Comment (mxmou) [reply addons/news-annoucements/userscript.js:1]: You can remove it, but many userscripts do this even if they don't need the objects.

**Review by mxmou — APPROVED** [review]

File: addons/news-annoucements/userscript.js (hunk 3 above)
Comment (Shluffy) [reply addons/news-annoucements/userscript.js:1]: I'll remove it then.

File: addons/news-annoucements/userscript.js [hunk 4]
```diff
@@ -0,0 +1 @@
+window.location.replace("https://scratch.mit.edu/discuss/5/");
```
Comment (FunctionalMetatable) [inline addons/news-annoucements/userscript.js:1]: Should still be enclosed in an exporting function.
```javascript
export default async function() {
  window.location.replace("https://scratch.mit.edu/discuss/5")
}
```

File: addons/news-annoucements/addon.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Redirect scratch.mit.edu/news to Announcements Forum",
+  "description": "Redirects the page scratch.mit.edu/news to the announcemnts forum.",
```
Comment (FunctionalMetatable) [inline addons/news-annoucements/addon.json:3]: `announcemnts` is spelt incorrectly

**Review by FunctionalMetatable — CHANGES_REQUESTED** [review]

**Comment by jeffalo** [comment]
I don't know if this is actually useful for anyone, but a general purpose redirect addon could be useful.

**Comment by Shluffy** [comment]
> I don't know if this is actually useful for anyone, but a general purpose redirect addon could be useful.

mxmou was saying easterEgg maybe, and it is fit for that

**Comment by FunctionalMetatable** [comment]
Is it ok if I merge this into another PR which merges both of these redirects? I'll add you as credit @Shluffy

**Comment by Shluffy** [comment]
@FunctionalMetatable Wouldn't this require a lot of settings then? i'm fine with it if I get credit though

**Comment by WorldLanguages** [comment]
Does Scratch still link to scratch.mit.edu/news anywhere? If not this looks too small to be added (and even if the answer is yes, it's questionable)

**Comment by RedGuy12** [comment]
I don't think so.
________________________________

**Comment by Shluffy** [comment]
@WorldLanguages no it does not, or as far as I know. If this was added in a big redirect addon as FunctionalMetatable was saying, then maybe. But otherwise, should I close this?

**Comment by gosoccerboy5** [comment]
Did you misspell "Announcements"? https://github.com/ScratchAddons/ScratchAddons/tree/600fc5ab4136b8668110a5e14eceab53a34493ad/addons/news-annoucements

**Comment by WorldLanguages** [comment]
Too small to be an addon

**Pull request closed without merging**